Navaids/FIXES_General RNAV.txt

```diff
@@ -1 +0,0 @@
-D113L N051.15.21.080 E000.19.48.360
```

Comment:
why did we remove this?

Reply:
Whoops, didn't mean to delete that, adding it back